test/addons-napi/test_dataview/test.js

```diff
@@ -10,5 +10,4 @@ const buffer = new ArrayBuffer(128);
 const template = Reflect.construct(DataView, [buffer]);
 
 const theDataview = test_dataview.CreateDataView(template);
-assert.ok(theDataview instanceof DataView,
-          'The new variable should be of type Dataview');
+assert.ok(theDataview instanceof DataView);
```

Comment:
I think it would be best to include the actual instanceof value in the error message here instead of just removing the message.

Reply:
@BridgeAR The value of `theDataView instanceof DataView` will reliably be  `true` or `false` and thus `false` if this fails.  What might be useful is displaying the value of `theDataView` so it can be obvious why that failed. If `theDataView` is `undefined` rather than some kind of complex object, that's useful info.